packages/2d/src/components/Txt.ts

```diff
@@ -92,6 +92,9 @@ export class Txt extends Shape {
     const y = box.y + box.height / 2;
     context.save();
     context.textBaseline = 'middle';
+    const clipPath = new Path2D();
+    clipPath.rect(box.x, box.y, box.width, this.getHeight());
+    context.clip(clipPath);
```

Comment:
Clipping should be done in the `draw` method. `drawText` is called for each text segment.

`this.computedSize()` should be used to acquire the bounding box, analogically to how `Layout` does it.